app/templates/admin/administer-user-modal.jade

```diff
@@ -76,11 +77,31 @@ block modal-body-content
           td= moment.timezone(new Date(parseInt(prepaid.id.substring(0, 8), 16) * 1000)).tz('America/Los_Angeles').format('l')
           td
             if prepaid.get('startDate')
-              = moment.timezone(prepaid.get('startDate')).tz('America/Los_Angeles').format('l')
+              if view.prepaidTableState && view.prepaidTableState[prepaid.id] === 'editMode'
+                - var controlId='startDate-'+prepaid.id
+                input(id= controlId type="date" name="startDate" value=moment.timezone(prepaid.get('startDate')).tz('America/Los_Angeles').format('YYYY-MM-DD'))
+              else
+                = moment.timezone(prepaid.get('startDate')).tz('America/Los_Angeles').format('l')
           td
             if prepaid.get('endDate')
-              = moment.timezone(prepaid.get('endDate')).tz('America/Los_Angeles').format('l')
-          td #{(prepaid.get('redeemers') || []).length} / #{prepaid.get('maxRedeemers') || 0}
+              if view.prepaidTableState && view.prepaidTableState[prepaid.id] === 'editMode'
+                - var controlId='endDate-'+prepaid.id
+                input(id= controlId type="date" name="endDate" value=moment.timezone(prepaid.get('endDate')).tz('America/Los_Angeles').format('YYYY-MM-DD'))
+              else
+                = moment.timezone(prepaid.get('endDate')).tz('America/Los_Angeles').format('l')
+          td
+            if view.prepaidTableState && view.prepaidTableState[prepaid.id] === 'editMode'
+              - var controlId='totalLicenses-'+prepaid.id
+              = (prepaid.get('redeemers') || []).length + ' / ' 
+              input(id=controlId type="number" name="totalLicenses" value=prepaid.get('maxRedeemers'), style="width:3em")
```

Comment:
we generally use 'px' unit for width, i would recommend using the same here as well